modules/replication/dockerimage-replication/deployspec.yaml

```diff
@@ -8,15 +8,19 @@ deploy:
         - mv linux-amd64/helm /usr/local/bin/helm
     build:
       commands:
+        - |
+          if ! aws s3api head-bucket --bucket "addf-metadata-${AWS_DEFAULT_REGION}-${AWS_ACCOUNT_ID}"; then
+            aws s3api create-bucket --bucket "addf-metadata-${AWS_DEFAULT_REGION}-${AWS_ACCOUNT_ID}" --region "${AWS_DEFAULT_REGION}" --create-bucket-configuration LocationConstraint=${AWS_DEFAULT_REGION}
+          fi
         - |
           if [[ -n ${ADDF_PARAMETER_NO_REPLICATE} ]]; then
```

Comment:
This is no longer needed as per the latest design, we deploy replications module only when we want to replicate images